pkg/component/observability/plutono/plutono_test.go

```diff
@@ -547,6 +547,27 @@ status:
 			}
 		)
 
+		checkDeployedResources := func(dashboardConfigMapName string, dashboardCount int) {
+			deployment := deploymentYAMLFor(values)
+			utilruntime.Must(references.InjectAnnotations(deployment))
+
+			Expect(manifests).To(ConsistOf(
```

Comment:
I wondered about that. If the `ExpectWithOffset` here fails it will end up providing a link to the following go line (depending on the test case ofc):
https://github.com/gardener/gardener/blob/0db8508a9841d05b8b9851a7831c525f8c190acd/pkg/component/observability/plutono/plutono_test.go#L624
I thought that this wouldn't be too helpful when checking for the source of the error.

Anyway, instead of `...WithOffset` functions, it is now better to make a call to `GinkgoHelper()` at the start of helper functions. So I could switch to that if you think it would be better.

Reply:
I see. 
TIL that there is such thing like `GinkgoHelper()` and it is the recommended thingy in the gomega guide.

I am fine with the usage of `GinkgoHelper()` in this occurrence. But maybe you can leave adapting the testing guide for another PR as most likely with it we would like to also adapt the existing tests which is now out of scope of this PR.